README.md

```diff
@@ -1 +1,35 @@
-# portfolio-1-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description of Portfolio.
+This is the "Portfolio" Project with just Html and Css
+
+## Built With
+
+Html and Css
+
+
+## Getting Started
+
+npm clone https://github.com/heintayzar-hm/portfolio-1-microverse.git
```

Comment:
- [OPTIONAL] I think there is a small typo here, It should be `git clone` instead of `npm clone` 💡